phase2/main.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+#
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import httplib2
+import os
+import threading
+import urllib
+
+from google.appengine.ext import ndb
+from google.appengine.api import users
+from google.appengine.api import memcache
+from google.appengine.ext.webapp import template
+from oauth2client.client import GoogleCredentials
+from apiclient.discovery import build
+import webapp2
+
+PROJECT_ID = 0  # your numeric project id
+
+# [START credentials_setup]
+# Set up the Prediction API service
+credentials = GoogleCredentials.get_application_default()
+credentials = credentials.create_scoped(
+    'https://www.googleapis.com/auth/prediction'
+    ' https://www.googleapis.com/auth/devstorage.full_control')
+SERVICES = threading.local()
+# [END credentials_setup]
+
+# dont change this
```

Comment:
How about:

# Don't change this. This is the fixed project ID for Prediction API hosted mode.